### ISSUE TITLE
Integrate a block into official symfony certification

### ISSUE THREAD

**Issue by ElectricMaxxx** [symfony/diversity#33]
As i followed a discussion about senseo's symfony certification, i realized that this would be a great place to teach diversity and habbits in a open source community. 

I know this workshop is part of a commercial company we do have no influence in. But we should at least suggest to introduce that topic into the certificate programm.

Points on Agenda:
- what does it mean to use a open source software
- how communicate with the community
- what is contribution? Why needed? 
- explain habbit of a diverse community (a theme we try to fill with content here)

**Comment by zairigimad** [comment]
I think that the documentation of symfony mention that clearly https://symfony.com/doc/current/contributing/community/reviews.html#be-constructive , it's a good idea to add this in the certification but finally not everybody going to take the certification.

**Comment by lsmith77** [comment]
yeah .. I think the issue here is that its hard to bring such "soft topics" into a standardized certification which is just done by "dump" forms and without a human involved in the review of the responses.

**Comment by ElectricMaxxx** [comment]
But, having more human inside that process would be good at all. I mean: we as a community are trying to get good behavior amoung the members. It would be really bad if somebody simply buys a certificate (even if there are bussiness constraints of senseo behind it) and got really bad behavior.

**Comment by lsmith77** [comment]
@nicolas-grekas what about mentioning these topics as part of the certification material, while dealing with any problems that may pop up via our normal CARE process?

**Comment by nicolas-grekas** [comment]
adding @javiereguiluz here as he is more specifically in charge of the certification than I am

**Comment by javiereguiluz** [comment]
Thanks for your proposal. However, although I'm in charge of the technical side of the certification, I can't make this kind of decisions about contents. Sorry!

**Comment by lsmith77** [comment]
Is there someone more appropriate to talk to? or is the timing just not ideal right now?
